ivy/functional/ivy/experimental/elementwise.py

```diff
@@ -1335,3 +1335,54 @@ def digamma(
     ivy.array([-0.7549271   0.92278427  0.9988394])
     """
     return ivy.current_backend(x).digamma(x, out=out)
+
+
+@handle_exceptions
+@handle_nestable
+@handle_out_argument
```

Comment:
```suggestion
```
No need for this as the out argument can be handled in the implementation directly.